packages/binding-coap/test/coap-server-test.ts

```diff
@@ -251,4 +253,88 @@ class CoapServerTest {
 
         return coapServer.stop();
     }
+
+    @test async "should support /.well-known/core"() {
+        const portNumber = 9001;
+        const coapServer = new CoapServer(portNumber);
+
+        await coapServer.start(null);
+
+        const testThing = new ExposedThing(null, {
+            title: "Test",
+            properties: {
+                test: {
+                    type: "string",
+                    forms: [],
+                },
+            },
+        });
+        testThing.setPropertyReadHandler("test", (options) => {
+            return new Promise<InteractionInput>((resolve, reject) => {
+                resolve(null);
+            });
+        });
```

Comment:
I think you can remove this read handler together with the TD test property for simplicity. 
```suggestion
      
```

Reply:
Oh, you are right! I removed the handlers and the unneeded properties :)